src/core/__tests__/chart-core-utils.test.tsx

```diff
@@ -22,17 +22,153 @@ describe("CoreChart: utils", () => {
     ["circle", { type: "scatter", symbol: "circle", options: {} }],
     ["circle", { type: "scatter", symbol: "unknown", options: {} }],
   ])('getSeriesMarkerType returns "%s" for series %j', (markerType, series) => {
-    expect(getSeriesMarkerType(series as any)).toBe(markerType);
+    expect(getSeriesMarkerType(series as Highcharts.Series | undefined)).toBe(markerType);
   });
 
-  test.each([getSeriesColor, getPointColor].map((fn) => [fn.name, fn] as const))(
```

Comment:
Now that getSeriesColor is excluded from this test, the cases like `getSeriesColor(undefined)` or `getSeriesColor({ color: {} })` are no longer covered.